pages/linux/deluser.md

```diff
@@ -0,0 +1,21 @@
+# deluser
+
+> Delete a user from the system.
+> Note: all commands must be executed as root.
+> More information: <https://manpages.debian.org/deluser.8.en.html>.
+
+- Remove a user:
+
+`deluser {{username}}`
+
+- Remove a user and their home directory:
+
+`deluser --remove-home {{username}}`
+
```

Comment:
Could you add an extra example for `--backup`:
```suggestion

- Remove a user and their home, but backup their files into a `.tar.gz` file in the current directory:

`deluser --backup --remove-home {{username}}`

```

Reply:
Actually @navarroaxel, it goes to `/` not the cwd.

Reply:
My mistake. Check the description now.

Reply:
How about just `in /`?

Reply:
> I tried playing with it, and I discovered that pure `--backup` doesn't work [...]

I got no absolutley no files in / when I was using it :shrug: Ubuntu 20.04